docs-new/docs/graphql/core/api-reference/relay-graphql-api/query.mdx

```diff
@@ -70,21 +72,15 @@ subscription {
 }q
 ```
 
-<div class="note">
+:::info Note
 
-<div class="title">
+For details of usage, please see [this page](/graphql/core/databases/postgres/schema/relay-schema.mdx).
 
-Note
-
-</div>
-
-For details of usage, please see `this page <pg_relay_schema>`.
-
-</div>
+:::
 
 ## Syntax definitions
 
-### Connection Object
+### Connection Object {#connection-object}
```

Comment:
pls remove space in label and remove hyphen in id

Reply:
Done